fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/Objects.java

```diff
@@ -10,6 +10,8 @@
   MEETING("meeting"),
   MESSAGE("message"),
   ROLL_CALL("roll_call"),
+  ELECTION_QUESTION("question"),
```

Comment:
Those objects (ELECTION_QUESTION, ELECTION_VOTE) are not required, as the corresponding classes are not of data type